src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -53,6 +53,7 @@ public partial class HubConnection
 
         // Transient state to a connection
         private ConnectionState _connectionState;
+        internal int _serverProtocolMinorVersion;
```

Comment:
This doesn't need to be `internal` any more, right?

Reply:
oops